### PR TITLE
make tests.py executable and add shebang

### PR DESCRIPTION
**Pull request by devkral**
use shebang and mark tests.py executable so you can execute it directly

### CODE REVIEW

File: tests.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#! /usr/bin/env python3
```
Comment (tclancy) [inline tests.py]: Shouldn't this be `#! /usr/bin/env python` instead of assuming python 3 availability and preference?

**Comment by devkral** [comment]
I think explicit is better and in three years python2 is unsupported:
see https://www.python.org/dev/peps/pep-0373/

**Comment by tclancy** [comment]
Understood, but this is a library, so we try to support as many use cases as possible. In addition, perhaps in the future some systems won't have separate binaries/ aliases named `python3` and may just return to `python`.

**Comment by devkral** [comment]
strange, removing the python version crashes the tests for python2.

**Comment by devkral** [comment]
local everything is fine

**Comment by tclancy** [comment]
Yes, it looks like all the tests have been silently failing for some time now. This looks good and raises the issue. All set here.

**Comment by tclancy** [comment]
Actually do you mind adding a second shebang line to see if we can sort out the failing tests?

```
# -*- coding: utf-8 -*-
```

**Pull request merged**